lib/src/component_declaration/component_base.dart

```diff
@@ -444,6 +444,18 @@ abstract class UiStatefulComponent<TProps extends UiProps, TState extends UiStat
   // ----------------------------------------------------------------------
 }
 
+class WarnOnModify<K, V> extends MapView<K, V> {
```

Comment:
This should be private [_(add an underscore to the beginning of the name)_](https://www.dartlang.org/guides/language/language-tour#important-concepts).